src/mygrad/nnet/initializers/normal.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+from mygrad import Tensor
+
+def normal(*shape, mean=0, std=1, dtype=np.float32, constant=False):
+    """ Initialize a :class:`mygrad.Tensor` by drawing from a normal (Gaussian) distribution.
+
+    Parameters
+    ----------
+    shape : Sequence[int]
+        The output shape.
+
+    mean : Real, optional (default=0)
+        The mean of the distribution from which to draw.
+
+    std : Real, optional (default=1)
+        The standard deviation of the distribution from which to draw.
+
+    dtype : data-type, optional (default=float32)
+        The data type of the output tensor; must be a floating-point type.
+
+    constant : bool, optional (default=False)
+        If `True`, the returned tensor is a constant (it
+            does not back-propagate a gradient).
+
+    Returns
+    -------
+    mygrad.Tensor, shape=`shape`
+        A Tensor, with values drawn from Ɲ(μ, σ²), where μ=`mean` and σ=`std`.
+    """
```

Comment:
I know this is pesky, but I do think we should strive to have an Examples section in each docstring - especially since students will be encountering this

Reply:
:sweat_smile: yeah I realized all of these were missing examples as I was going to bed :smile: will update them all